AppController/djinn.rb

```diff
@@ -2745,9 +2727,7 @@ def write_database_info()
   def update_firewall()
     all_ips = []
     @nodes.each { |node|
-      if !all_ips.include? node.private_ip
-        all_ips << node.private_ip
-      end
+      all_ips << node.private_ip unless all_ips.include? node.private_ip
```

Comment:
I think this is harder to parse as a one-liner due to the ambiguity:
`(all_ips << node.private_ip) unless all_ips.include? node.private_ip` vs `all_ips << (node.private_ip unless all_ips.include? node.private_ip)`.